### DISCUSSION TITLE
Harfbuzz for Embedded Systems

### DISCUSSION THREAD

**Discussion by mta452** [harfbuzz/harfbuzz#3438] [General] [answered]
I’m curious to learn about the status of harfbuzz for embedded systems. I know that C++ is highly portable language, but I’m not sure if that’s comparable to C. Basically I’m keen to know about the vision.

**Comment by ebraminio** [comment, marked as answer]
We've heard lots of success stories before which some were shared privately with me long time ago and I even did some minor tweaks for ease of use with `armcc`. This however specifically depends on how custom is your toolchain, if your targeting embedded system has a custom C compiler and lacks a decent C++ compiler, not only harfbuzz but many desktop C libraries aren't able to support that custom C compiler of such toolchains otherwise you shouldn't have a hard time to use harfbuzz. Btw guess you'll find https://github.com/harfbuzz/kharfbuzz and https://github.com/harfbuzz/harfbuzzjs interesting.
